lua/kubectl/views/jobs/init.lua

```diff
@@ -0,0 +1,80 @@
+local ResourceBuilder = require("kubectl.resourcebuilder")
+local buffers = require("kubectl.actions.buffers")
+local commands = require("kubectl.actions.commands")
+local definition = require("kubectl.views.jobs.definition")
+local informer = require("kubectl.actions.informer")
+local tables = require("kubectl.utils.tables")
+
+local M = {
+  builder = nil,
+}
+
+function M.View(cancellationToken, from_cronjob)
+  if M.builder then
+    M.builder = M.builder
+        :display(definition.ft, definition.display_name, cancellationToken)
+        :setCmd(definition.url, "curl")
+        :fetchAsync(function(builder)
+          builder:decodeJson()
+          informer.start(builder)
+          vim.schedule(function()
+            M.Draw(cancellationToken, from_cronjob)
+          end)
+        end)
+  else
+    M.builder = ResourceBuilder:new(definition.resource)
+        :display(definition.ft, definition.display_name, cancellationToken)
+        :setCmd(definition.url, "curl")
+        :fetchAsync(function(builder)
+          builder:decodeJson()
+          informer.start(builder)
+          vim.schedule(function()
+            M.Draw(cancellationToken, from_cronjob)
+          end)
+        end)
+  end
+end
+
+function M.Draw(cancellationToken, from_cronjob)
+  M.builder = M.builder
```

Comment:
Can we use state.filter  to do this instead?

Reply:
not really cause here I filter based on `ownerReference` not just any field.

Reply:
That's true, a job can have it's own name not tied to cronjob, looking awesome!